ui/src/app/metadata/rsd-metadata.enum.ts

```diff
@@ -1,10 +1,10 @@
 export enum MetadataType {
-  Category = "category",
-  Keyword = "keyword",
-  Standard = "standard",
-  Certification = "certification",
-  Alignment = "alignment",
-  Employer = "employer",
-  Author = "author",
-  JobCode = "jobcode"
+  Category = "categories",
+  Keyword = "keywords",
+  Standard = "standards",
+  Certification = "certifications",
+  Alignment = "alignments",
+  Employer = "employers",
+  Author = "authors",
+  Occupation = "occupations"
```

Comment:
Should this enum value be occupation or job code?

Reply:
Now is  `JobCode = "occupations" ` since job code is for code, and occupations is displayed in the view.
Is the conversation resolved with this change?